### PR TITLE
OSDOCS-10746: adds 4.14.29 relnote MicroShift

### PR DESCRIPTION
**Pull request by ShaunaDiaz**
Version(s):
4.14

Issue:
[OSDOCS-10746](https://issues.redhat.com/browse/OSDOCS-10746)

Link to docs preview:
[4.14.29 relnote](https://77155--ocpdocs-pr.netlify.app/microshift/latest/microshift_release_notes/microshift-4-14-release-notes.html#microshift-4-14-29-dp)

QE review:
- N/A stock text

### CODE REVIEW

**Review by mletalie — COMMENTED** [review]
/label peer-review-in-progress

**Review by mletalie — COMMENTED** [review]
/remove-label peer-review-needed
/remove-label peer-review-in-progress
/label peer-review-done
/lgtm

**Comment by mletalie** [comment]
One comment, otherwise LGTM!

File: microshift_release_notes/microshift-4-14-release-notes.adoc [hunk 1]
```diff
@@ -530,4 +530,15 @@ Issued: 2024-05-30
 
 {product-title} release 4.14.27 is now available. The list of enhancements and bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2024:3334[RHBA-2024:3334] advisory. The images that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2024:3331[RHSA-2024:3331] advisory.
 
+For the latest images included with {microshift-short}, view the contents of the `microshift-release-info` RPM. See xref:../microshift_install/microshift-embed-in-rpm-ostree-offline-use.adoc#microshift-embed-microshift-image-offline-deployment_microshift-embed-rpm-ostree-offline-use[Embedding {microshift-short} containers for offline deployments].
```
[the remark this answers is not in the document]
Comment (ShaunaDiaz) [reply microshift_release_notes/microshift-4-14-release-notes.adoc:533]: It is, you can see it in step 2. At some point we might make that a dedicated text snippet....

**Pull request merged**